### DISCUSSION TITLE
Installation Error with configuration file

### DISCUSSION THREAD

**Discussion by xgaibtcgd** [matt8707/hass-config#111] [General]
Hello,

So i have all the addons required installed and i'm trying to paste what's in the ui-lovelace.yaml since i'm not running in yaml mode.  I'm going into the raw configuration editor and i'm getting this error:

`Unable to parse YAML: YAMLException: unknown tag !<!include_dir_merge_named> (2:49) 1 | ... ates: 2 | ... rge_named button_card_templates -----------------------------------------^ 3 | ... 4 | ...`

Any idea what i might be doing wrong?

**Comment by xgaibtcgd** [comment]
Just doing some more reading it sounds like your home assistant has to be in yaml mode to do this install.  Is there a way to install the theme without being in yaml mode?

**Reply by matt8707** [reply]
it's because "include" doesn't work in ui.
you could technically paste all popups and button-card-templates into ui-lovelace (if you want to maintain a ~6000 lines file)

**Reply by IncByteLC** [reply]
Thx already found the issue. you are right. 
Changes to YAML mode - and everything works!
